sass/main.scss

```diff
@@ -66,31 +102,3 @@ hr {
     scrollbar-color: #333 #000;
   }
 }
-
-main {
```

Comment:
What is the reasoning for moving?

Reply:
`main` at the bottom of the file is illogical.

`@media (prefers-color-scheme: dark)` at the bottom is logical.